awx/main/models/credential/__init__.py

```diff
@@ -151,7 +151,7 @@ def needs_ssh_password(self):
     @property
     def has_encrypted_ssh_key_data(self):
         try:
-            ssh_key_data = decrypt_field(self, 'ssh_key_data')
+            ssh_key_data = self.get_input('ssh_key_data')
```

Comment:
I hate that we have to actually pull the value from the third party credential here (because it's *in* the HTTP request), but I can't think of any other way to do this.